aws-kendra-datasource/aws-kendra-datasource.json

```diff
@@ -1032,6 +1032,74 @@
         "IndexFieldName"
       ]
     },
+    "GoogleDriveConfiguration": {
+      "type": "object",
+      "properties": {
+        "SecretArn": {
+          "$ref": "#/definitions/SecretArn"
+        },
+        "InclusionPatterns": {
+          "$ref": "#/definitions/DataSourceInclusionsExclusionsStrings"
+        },
+        "ExclusionPatterns": {
+          "$ref": "#/definitions/DataSourceInclusionsExclusionsStrings"
+        },
+        "FieldMappings": {
+          "$ref": "#/definitions/DataSourceToIndexFieldMappingList"
+        },
+        "ExcludeMimeTypes": {
+          "$ref": "#/definitions/ExcludeMimeTypesList"
+        },
+        "ExcludeUserAccounts": {
+          "$ref": "#/definitions/ExcludeUserAccountsList"
+        },
+        "ExcludeSharedDrives": {
+          "$ref": "#/definitions/ExcludeSharedDrivesList"
+        }
+      },
+      "required": [
+        "SecretArn"
+      ]
+    },
+    "ExcludeMimeTypesList": {
+      "type": "array",
+      "minLength": 0,
+      "maxLength": 30,
+      "items": {
+        "$ref": "#/definitions/MimeType"
+      }
+    },
+    "MimeType": {
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 256
+    },
+    "ExcludeUserAccountsList": {
+      "type": "array",
+      "minLength": 0,
+      "maxLength": 100,
+      "items": {
+        "$ref": "#/definitions/UserAccount"
+      }
+    },
+    "UserAccount": {
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 256
+    },
+    "ExcludeSharedDrivesList": {
+      "type": "array",
+      "minLength": 0,
+      "maxLength": 100,
```

Comment:
don't think `minLength`/`maxLength` can be applied to `array` types:
https://github.com/aws-cloudformation/cloudformation-cli/issues/414
https://json-schema.org/understanding-json-schema/reference/array.html#length